corehq/tests/noseplugins/cmdline_params.py

```diff
@@ -3,6 +3,17 @@
 """
 from nose.plugins import Plugin
 
+REUSE_DB_HELP = """
+To be used in conjunction with the environment variable REUSE_DB=1.
+reset: Drop existing test dbs, then create and migrate new ones, but do not
+    teardown after running tests. This is convenient when the existing databases
+    are outdated and need to be rebuilt.
+flush: Flush all objects from the old test databases before running tests.
+    Much faster than `reset`.
+migrate: Migrate the test databases before running tests.
+    teardown: Skip database setup; do normal teardown after running tests.
```

Comment:
indentation error. Also, have you checked the format of this when printed with `--help`? Unfortunately I think argparse will indiscriminately remove all your nice indentation here, which is a shame. Not sure there is any better approach though :(